extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -364,7 +364,10 @@ void TailFile::onSchedule(const std::shared_ptr<core::ProcessContext> &context,
 
     context->getProperty(RecursiveLookup.getName(), recursive_lookup_);
 
-    context->getProperty(LookupFrequency.getName(), lookup_frequency_);
+    int64_t lookup_frequency;
```

Comment:
Can you leave a comment on this? If I were to encounter this, I would immediately think of reformatting it to the previous version.

Reply:
Done.  We should probably add a CI job which uses clang/libc++ at some point.